content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,57 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+The Palette allows the Pause and Resume of Palette Virtual Clusters when not in use. This feature enables the users with optimized resource sharing by pausing the virtual clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette virtual clusters. 
+
+<br />
+
+### Resource Dependencies
+
+* The Quota allocation is independent of a Virtual Cluster's Pause or Resume status.
+
+
+* The CPU and Memory are freed upon pausing a Virtual Cluster.
```

Comment:
Let's be specific, what CPU and memory is it freed? Is it from the overall user's quota?